_eo_031_enemies/orthobug.md

```diff
@@ -11,8 +11,10 @@ vulnerabilities:
   heavy: true
   sleep: unknown
   slow: true
-  stun: unknown
+  stun: true
 abilities:
   - name: Leech
-    description: "instant; drains HP"
+    potency: 160
+    type: Magic
+    description: 'instant; absorbs 74.8% of damage dealt'
```

Comment:
So specific 😆

Reply:
I'm sure this is a bug in the HP draining logic, and I filed a bug about the same thing in HoH, but I guess they didn't care enough to do anything about it... lol